index.html

```diff
@@ -157,8 +157,10 @@
         </script>
 
         <!-- The following statement is needed because the CCapture library seems to have not be written as an ES6-style module, so it can't be imported with an import statement -->
```

Comment:
Next update I hope to fix this, if nothing else I'll make a quick ES6 module wrapper for it